site/content/3.13/develop/http-api/databases.md

```diff
@@ -267,10 +278,10 @@ paths:
                       type: number
                 users:
                   description: |
-                    An array of user objects. The users will be granted *Administrate* permissions
-                    for the new database. Users that do not exist yet will be created.
+                    An array of user objects. The users is granted *Administrate* permissions
```

Comment:
```suggestion
                    An array of user objects. The user is granted *Administrate* permissions
```

Reply:
Should be plural because you can specify multiple users